README.md

```diff
@@ -27,25 +117,148 @@ mtx.myTranspose();
         [2, 4, 6]
        ]
 ```
+Not sure that this is what you wanted -
+```js
```

Comment:
We want to be able to use this for any mtx of any size. Try using a second for loop. Please refer to the solutions.